packages/core/src/Modules/Cashier/Containers/cashier.jsx

```diff
@@ -52,7 +52,9 @@ class Cashier extends React.Component {
 
         if (!response.error) this.setState({ is_p2p_restricted: false });
         else if (response.error.code !== 'RestrictedCountry') this.setState({ is_p2p_restricted: false });
```

Comment:
Not related to your change but could we merge these two if statements together with an `or` condition?
```suggestion
        if (!response.error || response.error.code !=== 'RestrictedCountry')
        this.setState({ is_p2p_restricted: false });
```

Reply:
@salar-fs your suggestion won't work because if there are no errors or if there is an error but it's not about restricted country then we want to set the restriction status to false. your suggestion only covers the second case.